### DISCUSSION TITLE
Newbie needs help

### DISCUSSION THREAD

**Discussion by kentezarmahdi** [aqualinkd/AqualinkD#278] [Q&A]
Hi, I am new to this community, but when I learned I could control my 20-year-old pool controller with a Raspberry Pi I jumped on it! Here is where I am stuck. I bought a Raspberry Pi 4 and connected the recommended USB2RS485 adapter to my panel and I performed the test and confirmed it is indeed connected to the panel. I then performed the "Simple install; trust everything." After the install, I didn't mess with any configuration files, because I do not know how to code. I have a RS8 panel, so I assumed the configuration as-is should work. 

After doing this, I tried to see if I could access the web UI, but I'm not sure if I'm reading it wrong or if I missed a step, but I cannot access the interface. The IP address for the raspberry pi is 10.0.0.93. I tried typing in all of the following into a web browser on a computer that is connected to the same network and all give me errors. 

http://aqualink.ip.address/ 
http://aqualink.10.0.0.93/ 
http://aqualink.ip.10.0.0.93/ 
http://aqualink.10.0.0.93.address/ 

Any help on what I should do next would be greatly appreciated.

**Comment by sfeakes** [comment]
Just the IP address.  so the below.

http://10.0.0.93/

If you used the "simple/trust everything" install and **_if_** your local network supports mdns, you can also use this.  Most tablets and phone support mdns btw.

http://aqualinkd.local/
(Replace aqualinkd with the machine name / host name of your pi)

**Reply by kentezarmahdi** [reply]
I tried both without any success. I tried with both my computer (mac) and iphone to no avail. I get the below error "This site can’t be reached10.0.0.93 refused to connect." My raspberry pi's host name is just raspberrypi, so I tried http://raspberrypi.local/ and I get the below error: "This site can’t be reachedraspberrypi.local refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED"

**Reply by sfeakes** [reply]
This is an issue with your network, or maybe how you have setup the pi. (If aqualinkd is running)

How are you connecting to your Pi?   Are you using ssh or a screen keyboard?
if ssh, use the same IP that your using for ssh.
If screen keyboard, then type in ifconfig to get the IP.  If it’s the one above (10.0.0.93) then try ‘curl http://10.0.0.93’  if you get an error, try ‘curl http://localhost’ and see if you get an error.

**Reply by kentezarmahdi** [reply]
I am connecting to Pi with ssh. I use the 10.0.0.93 to connect with ssh and it works. I tried both ‘curl http://10.0.0.93’ and 'curl http://localhost' and both take me to a google search of those words. I use a Netgear Orbi mesh network as my home network, I'm not sure if that has anything to do with it. I am connected to my pi via ssh using my macbook to access it and give commands through terminal, but no luck with any of the addresses I input into my browser (I tried chrome and safari).

**Reply by sfeakes** [reply]
On the pi within the shell type ```curl http://10.0.0.93``` and ```curl http://localhost```, if you get an error then AqualinkD is not listening on port 80, if you get a reply then something is up with the network.
There was a similar issue someone had on a google mesh network a few years back, and that was a limitation of wired and wireless routing.  But if that’s the case, I would expect ssh to fail as well.

**Reply by kentezarmahdi** [reply]
I get the below error when I enter curl http://10.0.0.93 or if I enter curl http://localhost into the terminal on the pi.

"curl: (7) Failed to connect to 10.0.0.93 port 80 after 0 ms: Couldn't connect to server"

**Reply by sfeakes** [reply]
Aqualinkd is not running on the Pi.
What’s the output of below?
`systemctl status aqualinkd`.

The below command will start aqualinkd,.
`systemctl start aqualinkd` 

But I just read your fist message again I overlooked the part where you said you **didn’t** do the configuration.  You have to modify /etc/aqualinkd.conf file, or it will fail to start.
If you run the following, it will tell you the main changes you need to make to the config file.
```
cd ~/software/AqualinkD
sudo ./release/serial_logger /dev/ttyUSB0
```

You can modify /etc/aqualinkd.conf with any editor, vim or vi are both installed on the pi by default.  Or if you prefer to edit the file with a desktop machine, you can also mount the pi file system in many different ways.  Here is a tutorial on that.
https://www.instructables.com/ShareEditCreate-Files-for-Your-Raspberry-Pi-From-a/
